.ci/openvino-onnx/watchdog/src/git_wrapper.py

```diff
@@ -94,15 +94,6 @@ def _get_pull_requests(self):
         :return:                    Paginated list of Pull Requests in GitHub repo
         :rtype:                     github.PaginatedList.PaginatedList of github.PullRequest.PullRequest
         """
-        message = "git: {}".format(self.git)
-        message2 = "repository: {}".format(self.repository)
-        message3 = "project: {}".format(self.project)
-        message4 = "github_credentials: {}".format(self.github_credentials)
-        log.warning(message)
-        log.warning(message2)
-        log.warning(message3)
-        log.warning(message4)
-
```

Comment:
Do we need this function since it does nothing?

Reply:
get_pull_requests return pull requests (line 97) :)
I just deleted warning logs added previously for debug purposes - currently those logs are unnecessary

Reply:
I can see it now, my browser window was not wide enough and it looked like an empty return...